### ISSUE TITLE
way to track what bosses you did

### ISSUE THREAD

**Issue by Goten87** [kevinclement/SimpleArmory#249]
a checkbox in the mount planning will be nice to see what we did already

so when i logout for the day and come back the next day i can load website again and see here i was so now i need to go to this raid

**Comment by jbryson3** [comment]
Until this PR has landed this is the way I do it:

1. Open up the developer tools
2. In the console type paste in the following jQuery code
```js
$('>tbody>tr>td:first-child', $('table').first()).each((i, el) => $(el).append('<input type="checkbox" />'))
```

**Comment by coogie** [comment]
Yeah, I got bored of waiting for this PR to land, so I build my own planner with this feature 😅
https://planner.overlazy.com

**Comment by czarl21** [comment]
> Yeah, I got bored of waiting for this PR to land, so I build my own planner with this feature 😅 https://planner.overlazy.com

I like it...it seems to pickup a few different mounts, but I like the checkmark idea.  Can you add the additional notes to it, like Icecrown in Heroic 25 and running the dungeons like Stonecore 10 times...it would also be nice to have a note if it is a raid (weekly), heroic (daily) or regular dungeon (hourly)

**Comment by JeromeM** [comment]
Here we are in 2025 and still no checkboxes?
And planner.overlazy.com isn't working anymore 😒
